src/bramble/client/main.js

```diff
@@ -806,6 +806,10 @@ define([
         this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_DESKTOP_PREVIEW"}, callback);
     };
 
+    BrambleProxy.prototype.toggleFullscreenPreview = function(callback) {
```

Comment:
Let's split this into `enableFullscreenPreview()` and `disableFullscreenPreview()` so that you can programmatically do what you want (i.e., manage fullscreen state in the hosting app vs having it be implicit in Bramble).